### ISSUE TITLE
PoB does not draw anything when run via Wine after recent commit

### ISSUE THREAD

**Issue by ahedgehogtf** [PathOfBuildingCommunity/PathOfBuilding#7402]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### How is Path of Building expected to behave?

When I open PoB, something should be drawn on the screen instead of remaining blank indefinitely.

### How does Path of Building behave?

When I open PoB via Wine on Linux from HEAD on the dev branch, a window is successfully created, but afterwards nothing gets drawn on it. It remains black with the window title "Download information not found". This is a new problem - I have run PoB on Linux via Wine for years from the dev branch without issue. I bisected the problem to commit `01c988184fd95442d54e2ddc812d5ff27e08212d`. Commits before this one run just fine.

### How to reproduce the issue

1. `git checkout 01c988184fd95442d54e2ddc812d5ff27e08212d`
2. `wine runtime/Path\{space\}of\{space\}Building.exe`
3. Observe nothing being drawn

Wine startup logs:
```
0138:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
01cc:fixme:d3dcompiler:D3DCompile2 Ignoring flags 0xc000.
01c4:fixme:d3dcompiler:D3DCompile2 Ignoring flags 0xc000.
vkd3d:01c4:fixme:hlsl_yylex Unsupported preprocessor #pragma directive at line 11.
vkd3d:01cc:fixme:hlsl_yylex Unsupported preprocessor #pragma directive at line 12.
01dc:fixme:d3dcompiler:D3DCompile2 Ignoring flags 0xc000.
01d8:fixme:d3dcompiler:D3DCompile2 Ignoring flags 0xc000.
vkd3d:01dc:fixme:hlsl_yylex Unsupported preprocessor #pragma directive at line 7.
vkd3d:01d8:fixme:hlsl_yylex Unsupported preprocessor #pragma directive at line 8.
01ec:fixme:d3dcompiler:D3DCompile2 Ignoring flags 0xc000.
01f0:fixme:d3dcompiler:D3DCompile2 Ignoring flags 0xc000.
vkd3d:01f0:fixme:hlsl_yylex Unsupported preprocessor #pragma directive at line 9.
vkd3d:01ec:fixme:hlsl_yylex Unsupported preprocessor #pragma directive at line 8.
```

I am using Wine 9.1 on Fedora 39.

**Comment by zao** [comment]
The major change now is that `dev` now uses my new 64-bit ANGLE-based renderer rather than the old 32-bit OpenGL 1.1 renderer. It has been available in the `beta` channel of the regular PoB releases since fall 2023.

Last time I tested it under Wine on Linux and macOS, I also needed to run `winetricks d3dcompiler_47` to get a properly functioning shader compiler rather than the incomplete one Wine has out of the box. Give that a shot.
